src/booleans.js

```diff
@@ -1,77 +1,133 @@
 const negate = (a) => {
-  // your code here
-};
+    if (a === true) {
```

Comment:
`a` will evaluate to either a truthy or a falsy value. You can use the `!` operator to flip it's truthy/falsy status which would cut out the need for checking for a strictly true value. 

Also, you can return an expression - there isn't a need for `if/else` here:

```js
return !a
```